boto3/s3/inject.py

```diff
@@ -13,12 +13,40 @@
 from boto3.s3.transfer import S3Transfer
 from boto3 import utils
 
+from botocore.exceptions import ClientError
 
 def inject_s3_transfer_methods(class_attributes, **kwargs):
     utils.inject_attribute(class_attributes, 'upload_file', upload_file)
     utils.inject_attribute(class_attributes, 'download_file', download_file)
 
 
+def inject_bucket_load(class_attributes, **kwargs):
+    utils.inject_attribute(class_attributes, 'load', bucket_load)
+
+
+def bucket_load(self, *args, **kwargs):
+    """Fake s3.Bucket.load method.
```

Comment:
If this is a custom method, I will use this docstring because it is not modeled in the resource model.

I have been auto-documenting the ``load`` and ``reload`` methods as the following:
```
Calls [service].Client.[method]() to update the attributes of the [name_of_resource] resource.
```
So maybe do it as?
```
Calls s3.Client.list_objects() to update the attributes of the Object resource.
```
Then you can move the original docstring to a comment or into the inject method.

Let me know what you think.